presto-docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -23,12 +23,14 @@ Configuration
 -------------
 
 To configure the Black Hole connector, create a catalog properties file
-``etc/catalog/blackhole.properties`` with the following contents:
+``/etc/opt/prestoadmin/connector/blackhole.properties`` with the following contents:
```

Comment:
there are a couple of other places to change, but I'll add a commit to your branch with them